### DISCUSSION TITLE
export data into .bag file on cfclient?

### DISCUSSION THREAD

**Discussion by ugrm** [bitcraze/discussions#1416] [Q&A]
I'm trying to export the multiranger data from the plotter into a .bag file for the google cartographer. However, I can only find csv, images, mattplotlib, and svg file formats. Is there a way to get a .bag file directly from the cfclient? If not, are there ways to turn the mentioned files into a .bag file?

**Comment by knmcguire** [comment]
Hi! I reopened your discussion since this is useful for more people. We don't have functionality for .bag for the cfclient, so this need to be implemented as a new feature, or that you write a script yourself with the cflib that does that for you.

**Reply by ugrm** [reply]
I see, if I make any progress regarding this part, I'll upload it
